integreat_cms/cms/templates/push_notifications/push_notification_list_row.html

```diff
@@ -18,9 +19,7 @@
 				{% spaceless %}
 					{% for other_language in languages %}
 					<a href="{% url 'edit_push_notification' push_notification_id=push_notification.id region_slug=region.slug language_slug=other_language.slug %}">
-						{% with pnt=push_notification|translation:other_language %}
-						<i data-feather="{% if pnt %}edit-2{% else %}plus{% endif %}" class="text-gray-800"></i>
-						{% endwith %}
+						<i data-feather="{% if other_language in existing_languages %}edit-2{% else %}plus{% endif %}" class="text-gray-800"></i>
```

Comment:
```suggestion
						<i data-feather="{% if other_language in push_notification.languages %}edit-2{% else %}plus{% endif %}" class="text-gray-800"></i>
```
Once the property is cached, there is no reason anymore to wrap it into a `{% with %}` tag.